code/modules/uplink/uplink_item.dm

```diff
@@ -452,12 +452,6 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/ammo_box/magazine/sniper_rounds/penetrator
 	cost = 5
 
-/datum/uplink_item/ammo/sniper/accelerator
```

Comment:
http://www.ss13.eu/tgdb/tg/latest_stats.html#uplinkitems

Literally never bought, and I want to use the same code for this